data/commodities.txt

```diff
@@ -395,6 +424,7 @@ trade
 		"autoclaves"
 		"bandages"
 		"battlefield trauma kits"
+		"beds"
```

Comment:
I'm not sure if `beds` really make sense in the medical list. We've already got `hospital beds`, and `beds` sounds more like furniture than medical equipment.

Reply:
Deleted in ae326a6 .